Source/Scene/Model.js

```diff
@@ -772,6 +781,10 @@ define([
         return JSON.parse(json);
     }
 
+    Model.getDefaultCacheKey = function(url) {
```

Comment:
When making the cache key `Model` converts the url to an absolute url, otherwise I would use `_url` directly. I'm handling the case where `ModelInstanceCollection` creates the `Model` from JSON or binary now.